src/main/java/org/synyx/urlaubsverwaltung/absence/AbsenceApiController.java

```diff
@@ -160,7 +163,14 @@ private List<AbsenceDto> toAbsenceDto(AbsencePeriod.Record absenceRecord) {
     private AbsenceDto toAbsenceDto(LocalDate date, DayLength dayLength, AbsencePeriod.RecordInfo recordInfo) {
         final AbsenceDto.AbsenceType type = toAbsenceTypes(recordInfo.getAbsenceType());
         final String status = recordInfo.getStatus().name();
-        return new AbsenceDto(date, type, recordInfo.getId().orElse(null), status, dayLength, recordInfo.getCategory().orElse(null), recordInfo.getTypeId().orElse(null));
+        final String category = recordInfo.getCategory().orElse(null);
+        AbsenceDto absenceDto = new AbsenceDto(date, type, recordInfo.getId().orElse(null), status, dayLength, category, recordInfo.getTypeId().orElse(null));
+
+        Optional<Long> absenceId = recordInfo.getId();
+        if (VacationCategory.OVERTIME.name().equals(category) && absenceId.isPresent()) {
+            absenceDto.add(linkTo(methodOn(OvertimeAbsenceApiController.class).overtimeAbsence(recordInfo.getPerson().getId(), absenceId.get())).withRel("usedOvertime"));
```

Comment:
siehe auch Diskussion zum Attributs name des DTOs.

Das ist eine Abwesenheit vom Typ Überstunden. Also "abgebaute Überstunden"? -> `reducedOvertime`

Vielleicht reicht hier aber auch einfach nur `overtime` als Referenz? Ich bekomme eine Abwesenheit, möchte mehr Informationen haben wie z. B. die `duration`. In dem fall hier ist sind es Überstunden 🤔

Reply:
Hab `overtime? als Referenz übernommen